### ISSUE TITLE
A runtime error occurred during the loading of the COM Add-in.

### ISSUE THREAD

**Issue by PGS62** [sdllc/Basic-Excel-R-Toolkit#134]
Dear Duncan

I can't get BERT 2.3.10 to work on a PC on which BERT 1.63 was working correctly. The PC is Windows 10, 64 bit and Office 2016, 64 bit. I ran an Office Repair and rebooted the PC to no avail.

Under:
File > Options > Add-ins > COM Add-ins > Go > BERT2 Ribbon Menu

I see:
Load Behavior: Not loaded. A runtime error occurred during the loading of the COM Add-in.

This is all very similar to #101, the resolution of which was (I think) provided by changes you made in BERT 2.2.1. Unfortunately this time BERT 2.2.1 does not install correctly either. After running the BERT installer, on first attempt Excel does not launch. On second attempt, a message box appears "Excel couldn't start last time...Do you want to start in safe mode?"

Do you think you might have a solution?

Best wishes

Philip

**Comment by duncanwerner** [comment]
Can you try 2.4.2, which is the most recent release? 
https://github.com/sdllc/Basic-Excel-R-Toolkit/releases/tag/2.4.2

Also make sure you're installing as a regular user, and not installing to a privileged directory.

**Comment by PGS62** [comment]
I downgraded my Windows account from Administrator to Standard User and installed BERT 2.4.2 to the default location c:\Users\<username>\AppData\Local\BERT2\. BTW I'm not  sure what you mean by privileged directory.

No success - still see "Not loaded. A runtime error occurred during the loading of the COM Add-in."

As a test, I tried installing onto the same "problem" PC under another user account. No dice. But installing on a second PC works fine of course.
